### ISSUE TITLE
Hi, why the loss function trained is negative

### ISSUE THREAD

**Issue by Reas2020** [XRad-Ulm/Proto-Caps#2]
I tried to run this part of the code and I found that val loss is negative, I tried to understand the loss function but I I still can't understand why. For example with Epoch 100, val acc = 0.9778 and val loss = -1.21901
